package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "is-mergeable",
-	"version": "1.2.0",
+	"version": "1.3.0-pagination.0",
```

Comment:
Can you undo this

Reply:
removed, agreed not sure why I included this.

Reply:
I assume you mean from the commit, not also the published version?

Reply:
correct